evm/src/cpu/kernel/interpreter.rs

```diff
@@ -123,13 +127,20 @@ impl<'a> Interpreter<'a> {
             0xa2 => todo!(),                                           // "LOG2",
             0xa3 => todo!(),                                           // "LOG3",
             0xa4 => todo!(),                                           // "LOG4",
+            0xa5 => panic!("Executed PANIC"),                          // "PANIC",
```

Comment:
I don’t think a program panicking should cause the _interpreter_ to panic. A panicking program is a perfectly valid input. Maybe consider returning a `Result`?

Reply:
Hm fair point. Changed it to return a `Result`.